code/game/machinery/kitchen/smartfridge.dm

```diff
@@ -230,6 +230,21 @@
 
 	RefreshParts()
 
+/obj/machinery/smartfridge/drinks/filled/New()
```

Comment:
condiments go in the base smartfridge, not the drink fridge (see accepted types list)

Reply:
```/obj/machinery/smartfridge/drinks
	name = "\improper Drink Showcase"
	desc = "A refrigerated storage unit for tasty tasty alcohol."

	accepted_types = list(	/obj/item/weapon/reagent_containers/glass,
							/obj/item/weapon/reagent_containers/food/drinks,
							/obj/item/weapon/reagent_containers/food/condiment)```

Reply:
Not that it matters. I'll make another PR to completely remove the drink fridge because the only difference is that the regular smart fridge doesn't hold glasses... Seems kinda pointless to have a completely different object just to insert glasses

Reply:
I kinda want to do it with this PR though